research/conditional/train/configs/baselines/bert_medium.yaml

```diff
@@ -0,0 +1,40 @@
+# grid args
+runner: "research.conditional.train.cc_train"
+time: "0-02:00:00"
```

Comment:
czy 2h serio wystarczają?

Reply:
Nie, to pomyłka. Ustawiłem teraz na 2 dni.